src/components/Filters/FilterChecklist.vue

```diff
@@ -48,16 +48,28 @@
           <LicenseIcons v-if="filterType === 'licenses'" :license="item.code" />
           {{ itemLabel(item) }}
         </label>
-        <img
+        <svg
           v-if="filterType === 'licenses'"
+          width="24"
+          height="20"
+          viewBox="0 0 24 20"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
           :aria-label="$t('browse-page.aria.license-explanation')"
           tabindex="0"
-          src="@/assets/help_icon.svg"
           alt="help"
           class="license-help pr-1"
           @click.stop="toggleLicenseExplanationVisibility(item.code)"
           @keyup.enter="toggleLicenseExplanationVisibility(item.code)"
-        />
+        >
+          <circle cx="12" cy="10" r="8" stroke="#1E1E1E" stroke-width="1.5" />
+          <path
+            d="M9.75 8.25C9.75 7.00736 10.7574 6 12 6C13.2426 6 14.25 7.00736 14.25 8.25C14.25 9.40828 13.3748 10.3621 12.2496 10.4863C12.1124 10.5015 12 10.6119 12 10.75V12"
+            stroke="#1E1E1E"
+            stroke-width="1.5"
+          />
+          <path d="M12 13V14.5" stroke="#1E1E1E" stroke-width="1.5" />
+        </svg>
```

Comment:
Because this is an interactive element, it would be best to wrap it in a `<button></button>` with the `.appearance-none` class. Please move the label, click and key events, etc. to the parent button. 

I wouldn't have expected a contributor to add this by the way, it's a bit of an extra request 😄

Reply:
Makes sense to me! I can certainly add that!

Reply:
@zackkrida When I group the `<label class="checkbox">` and the `<svg>` into the same `<button>` it groups them together into a cluster and it doesn't seem quite right.  I could style them of course to go back to the same formatting, but it seems a little odd to me that they'd all be in one button anyway.  Would it be better to put each individual interactive element into its own `<button>` tag?  Maybe I just misunderstood the request.

With the `<label>` and `<svg>` inside the same `<button>`:
![button-example](https://user-images.githubusercontent.com/82423961/134057753-5dfcd87b-bd0e-412c-bf6d-80b4bfcbdbf6.png)

Reply:
Ah yes, I apologize, I was just referring to moving the `aria-label` prop on the svg over to the button element!